src/Dropdown.js

```diff
@@ -27,8 +27,8 @@ const Dropdown = styled(DropdownBase)`
 `
 
 const DropdownCaret = styled.div`
-  border: ${get('space.1')} solid transparent;
-  margin-left: ${get('space.1')};
```

Comment:
These are values that are really specific to this component and we probably don't want updating automatically if we update the spacing scale, so I changed them to hardcoded values. I did this a few other places around the codebase as well.